src/bindings/c/docs/mapping_relationship_of_interfaces.md

```diff
@@ -0,0 +1,136 @@
+# Mapping Relationship of Interfaces
+
+Here provides the details mapping relationship between C API and C++ API in OpenVINO including components `core`, `model`, `compiled model`, `infer request`, `partial shape`, `prepostprocess`, `tensor`. This introduction doesn't provide all interfaces' mapping relationship from C++ to C in OpenVINO, but the most important & common interfaces provided.
+
+**NOTE**: For more matching details can be found in [Mapping Relationship of Interfaces](./docs/mapping_relationship_of_interfaces.md).
+
+## Contents:
+
+ - [The mapping relationship for core](#the-mapping-relationship-for-core)
+ - [The mapping relationship for model](#the-mapping-relationship-for-model)
+ - [The mapping relationship for compiled model](#the-mapping-relationship-for-compiled-model)
+ - [The mapping relationship for infer request](#the-mapping-relationship-for-infer-request)
+ - [The mapping relationship for partial shape](#the-mapping-relationship-for-partial-shape)
+ - [The mapping relationship for prepostprocess](#the-mapping-relationship-for-prepostprocess)
+ - [The mapping relationship for tensor](#the-mapping-relationship-for-tensor)
+ - [See also](#see-also)
+
+## The mapping relationship for core
+
+Here is the mapping relationship for core related,
+ * All core related APIs implemented with C++ can be found [here](../../../inference/include/openvino/runtime/core.hpp)
+ * All core related APIs implemented with C can be found [here](../include/openvino/c/ov_core.h)
+
+| C++ API | C API | Description |
```

Comment:
These tables are hard to maintain.
Shouldn't Reference API documentation answer on the same questions?